pkg/config/config.go

```diff
@@ -21,5 +21,7 @@ type CliConfig struct {
 
 // CreateClusterCLIConfig is the config we use for create cluster specific configurations.
 type CreateClusterCLIConfig struct {
-	SkipCPIPCheck bool
+	SkipCPIPCheck           bool
+	NodeStartupTimeout      string
+	UnhealthyMachineTimeout string
```

Comment:
Curious, do we currently let users modify the MHC timeouts during cluster upgrades (with the CLI)?

Reply:
no we don't, looks like that was another bug that i ended up fixing in this PR.